src/app/dao/tests/ChatDAO.spec.ts

```diff
@@ -19,7 +19,7 @@ import { UserDAO } from '../UserDAO';
 import { ChatService } from 'src/app/services/ChatService';
 import { Subscription } from 'rxjs';
 
-describe('ChatDAO', () => {
+xdescribe('ChatDAO', () => {
```

Comment:
`git add -a` is forbidden!

Reply:
good, I never do it !
Mais je garderais git add punt, c'pas le moyen âge ici !
Ces tests n'ont qu'à pas prendre douze minutes pour finalement décider que personne ne verra combien de code est couvert ! Na :D !

Reply:
ah oui `git add .` devrait être interdit sans `-p` ! 
ÇA FONCTIONNE TRÈS BIEN ICI MR ! Sur deux PC différents ainsi que sur les PC de github, donc tu dois CHANGER DE PC

Reply:
C'est rigolo parce que la vérité c'est l'inverse !

Reply:
!(changer de pc && résoudre) = !changer de pc || !résoudre ?!

Reply:
wat

Reply:
ton message précédent ne compile pas car "?" n'est pas une instruction en binaire !